pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -715,6 +716,20 @@ func GetOpenshiftBootstrapClusterRoles() []rbacv1.ClusterRole {
 				rbacv1helpers.NewRule("get", "put", "update", "delete").URLs(templateapi.ServiceBrokerRoot + "/*").RuleOrDie(),
 			},
 		},
+		{
+			ObjectMeta: metav1.ObjectMeta{Name: "resource-metrics-view",
+				Labels: map[string]string{"rbac.authorization.k8s.io/aggregate-to-view": "true", "rbac.authorization.k8s.io/aggregate-to-admin": "true"}},
```

Comment:
This should only aggregate into `view`.

Reply:
> This should only aggregate into view.

I don't think 3.11 has the aggregate the aggregated roles trick.